bentoml/_internal/io_descriptors/text.py

```diff
@@ -139,16 +138,29 @@ async def to_http_response(self, obj: str, ctx: Context | None = None) -> Respon
             return Response(obj, media_type=MIME_TYPE)
 
     async def from_grpc_request(
-        self,
-        request: _service_pb2.Request,
-        context: BentoServicerContext,  # pylint: disable=unused-argument
+        self, request: pb.Request, context: BentoServicerContext
     ) -> str:
-        return str(request.input.string_value)
+        import ast
+
+        from ..utils.grpc import get_field
+        from ..utils.grpc import validate_content_type
+
+        # validate gRPC content type if content type is specified
+        validate_content_type(context, self)
+
+        field = ast.literal_eval(get_field(request, self))
```

Comment:
why do we need this?

Reply:
You can an put a bytes string.

Reply:
We should also allow a bytes field in all of the message.